config.example.yaml

```diff
@@ -4,28 +4,28 @@ hosts:
     type: ssh
     username: root 
     password: somethingSecret
-  x.example.net:
-  y.example.net:
-  z.example.net:
-  "192.0.1.5":
-    alias: home-server
-    connection: 
-      type: ssh
-      username: root 
-      password: somethingSecret
-    port: 33
-  "192.0.1.4":
-    connection: 
-      type: local
   children:
+    x.example.net:
+    y.example.net:
+    z.example.net:
+    "192.0.1.5":
+      alias: home-server
+      connection: 
+        type: ssh
+        username: root 
+        password: somethingSecret
+        port: 33
+    "192.0.1.4":
+      connection: 
+        type: local
     eu-west1:
```

Comment:
What's `eu-west1` supposed to be in this context?

Reply:
@deven96 a group. You can create groups that share similar ssh configuration.

Reply:
Why is it different from the group that you just put them directly under children

Reply:
I don't understand

Reply:
lol... nevermind na shayo I shayo that night of this comment